tracing-appender/src/rolling.rs

```diff
@@ -1443,19 +1132,19 @@ mod test {
             "[year]-[month]-[day] [hour]:[minute]:[second] [offset_hour \
          sign:mandatory]:[offset_minute]:[offset_second]",
         )
-        .unwrap();
+            .unwrap();
```

Comment:
Maybe you need to run `cargo fmt`

Reply:
Right, the editor keeps resetting the auto-fmt option when I create a new project. Thanks 🙏